ci/docker/snaper/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# build image: docker build --no-cache -f ci/docker/snaper/Dockerfile --tag ghcr.io/nordsecurity/nordvpn-linux/snaper:0.0.1 .
+# build snap: docker run -v "$PWD":/build -w /build ghcr.io/nordsecurity/nordvpn-linux/snaper:0.0.1 snapcraft
+# cleanup: sudo rm -rf ./parts ./stage ./prime
+
+ARG RISK=stable
+# ubuntu 22.04
+ARG UBUNTU=jammy
+
+FROM ubuntu:$UBUNTU as builder
+ARG RISK
+ARG UBUNTU
+RUN echo "Building snapcraft:$RISK in ubuntu:$UBUNTU"
+
+# Grab dependencies
+RUN apt-get update \
+        && apt-get dist-upgrade --yes \
+        && apt-get install --yes curl jq squashfs-tools 
+
+# Grab the core snap (for backwards compatibility) from the stable channel and
+# unpack it in the proper place.
+RUN curl -L $(curl -H 'X-Ubuntu-Series: 16' 'https://api.snapcraft.io/api/v1/snaps/details/core' | jq '.download_url' -r) --output core.snap \
+        && mkdir -p /snap/core \
+        && unsquashfs -d /snap/core/current core.snap
+
+# Grab the core18 snap (which snapcraft uses as a base) from the stable channel
+# and unpack it in the proper place.
+RUN curl -L $(curl -H 'X-Ubuntu-Series: 16' 'https://api.snapcraft.io/api/v1/snaps/details/core18' | jq '.download_url' -r) --output core18.snap \
+        && mkdir -p /snap/core18 \
+        && unsquashfs -d /snap/core18/current core18.snap
+
+# Grab the core20 snap (which snapcraft uses as a base) from the stable channel
+# and unpack it in the proper place.
+RUN curl -L $(curl -H 'X-Ubuntu-Series: 16' 'https://api.snapcraft.io/api/v1/snaps/details/core20' | jq '.download_url' -r) --output core20.snap \
+        && mkdir -p /snap/core20 \
+        && unsquashfs -d /snap/core20/current core20.snap
```

Comment:
```suggestion
```
I think everything can be stripped which is not `core22`

Reply:
Right, it can be removed now. Initially had some issues with those. Done.